packages/nxtcm-rosa-hcp-wizard/src/components/PopoverHintWithTitle.css

```
@@ -15,3 +15,9 @@
     padding: 0;
     color: inherit !important
 }
+
+.pf-v6-c-content--pre:has(.pf-v6-c-clipboard-copy) {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)